src/test/kotlin/no/nav/medlemskap/regler/funksjoner/ArbeidsforholdFunksjonerTest.kt

```diff
@@ -187,6 +188,59 @@ class ArbeidsforholdFunksjonerTest {
         assertFalse(arbeidsforhold.harBrukerJobbetMerEnnGittStillingsprosentTilEnhverTid(25.0, kontrollperiode, Ytelse.SYKEPENGER))
     }
 
+    @Test
+    fun `Arbeidsforhold med både statlig og privat enhetstype får true`() {
+        val arbeidsforhold =
+            listOf(arbeidsforholdMedStatligJuridiskEnhetstype, arbeidsforholdMedIkkeStatligEllerKommunalJuridiskEnhetstype)
+
+        val kontrollperiode = Kontrollperiode(
+            fom = LocalDate.of(2019, 1, 1),
+            tom = LocalDate.of(2020, 1, 1)
+        )
+
+        val sjekkJuridiskEnhet = erArbeidsforholdetStatligEllerKommunalt(arbeidsforhold, kontrollperiode)
+        assertTrue(sjekkJuridiskEnhet)
+    }
+
+    @Test
+    fun `Arbeidsforhold med juridisk enhetstype som null får false`() {
+        val arbeidsforhold = listOf(arbeidsforholdMedFlerEnn6Ansatte)
+
+        val kontrollperiode = Kontrollperiode(
+            fom = LocalDate.of(2019, 1, 1),
+            tom = LocalDate.of(2020, 1, 1)
+        )
+
+        val sjekkJuridiskEnhet = erArbeidsforholdetStatligEllerKommunalt(arbeidsforhold, kontrollperiode)
```

Comment:
smak og behag, men jeg ville nok ha droppa å ha egen variabel på det her og puttet rett inn i assert

Reply:
Jeg føler at hvis man fôrer større linjer inn i en funksjon så kan koden ganske raskt bli uleselig og vanskelig å følge med på logikken bak. Ved å legge linjene til variabler håper jeg å gjøre det litt klarere hva som skjer via variabel navnet osv.

Folk med mer erfaring har kanskje ikke samme vansken som jeg har da 🙂

Reply:
Skjønner. Helt greit å beholde det, når du tydeligvis har reflektert over det, for det blir litt smak og behag.